tests/test_sprites.py

```diff
@@ -326,18 +326,3 @@ def test_rotatable_base_sprite():
 
     test_sprite.rotate(1)
     assert test_sprite.rotation == 1
-
-
-def test_sprite_in_main():
```

Comment:
Is there a solid way to keep a test around this? The use case of "no actual __file__" is still a possibility, we just resolve it differently.